spec/message_configuration_client_spec.rb

```diff
@@ -193,4 +196,22 @@
       expect(subject.send(:fetch_truststore_from_server)).to be_nil
     end
   end
+
+  describe "#restart_evmserverd" do
+    it "restarts evmserverd if it is running" do
+      expect(subject).to receive(:say)
+      evmserverd = double(@spec_name, :running? => true)
+      expect(evmserverd).to receive(:restart)
+      expect(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(evmserverd)
+      expect(subject.send(:restart_evmserverd)).to be_nil
+    end
+
+    it "does not restart evmserverd if it is not running" do
+      expect(subject).to receive(:say)
+      evmserverd = double(@spec_name, :running? => false)
```

Comment:
```suggestion
      evmserverd = LinuxAdmin::Service.new("evmserverd")
      expect(evmserverd).to receive(:running?).and_return(false)
```

Reply:
@bdunne Thank you for the help. I believe it is good now.